doc/nrf/ug_nrf9160_gs.rst

```diff
@@ -0,0 +1,502 @@
+  See `iBasis IoT network coverage`_ for information about which mode is supported where.
+  If your location has coverage for both, you can choose either one.
+
+Application firmware for Device Firmware Update (DFU)
+  The images in the :file:`img_fota_dfu_bin` and :file:`img_fota_dfu_hex` folders contain firmware images for DFU.
+  When following this guide, you can ignore these images.
+
+Modem firmware
+  The modem firmware is in a zip archive instead of a folder.
+  The zip is named :file:`mfwnrf9160_` followed by the firmware version number.
+  Do not unzip this file.
+
+.. _nrf9160_gs_updating_fw_application:
+
+Updating the application firmware
+=================================
+
+To update the application firmware, complete the following steps:
+
+1. Open the Programmer app.
+#. Make sure the **PROG/DEBUG SW10** switch on the nRF9160 DK is set to **nRF91**.
+   On DK v0.9.0 and earlier, this is the **SW5** switch.
+#. Connect the nRF9160 DK to the computer with a micro-USB cable, and then turn the DK on.
+#. Click :guilabel:`Select device` and select the DK from the drop-down list.
+   You can identify the nRF9160 DK by the fact that it has three COM ports when you expand its entry.
+
+   .. figure:: /images/programmer_com_ports.png
+      :alt: Programmer - COM ports
+
+      Programmer - COM ports
+
+   If the three COM ports are not visible, press ``Ctrl+R`` in Windows or ``command+R`` in macOS to restart the Programmer application.
+
+   The button text changes to the type of the selected device, with the SEGGER ID below the name.
+   The :guilabel:`Device Memory Layout` section also changes its name to the device name, and indicates that the device is connected.
+   If the :guilabel:`Auto read memory` option is selected in the :guilabel:`Device` section, the memory layout will update.
+   If it is not selected and you wish to see the memory layout, click :guilabel:`Read` in the :guilabel:`Device` section.
+
+#. Click :guilabel:`Add file` in the :guilabel:`File` section, and select :guilabel:`Browse`.
+
+   .. figure:: /images/programmer_addfile_nrf9160dk.png
+      :alt: Programmer - Add file
+
+      Programmer - Add file
+
+#. Navigate to where you extracted the firmware, and then to the :file:`img_app_bl` folder there.
+#. Select either :file:`nrf9160dk_asset_tracker_v2_ltem_<version-number>.hex` (LTE-M mode) or :file:`nrf9160dk_asset_tracker_v2_nbiot_<version-number>.hex` (NB-IoT mode), depending on where you are located.
+   See `iBasis IoT network coverage`_ for information about which mode is supported where.
+
+   For NB-IoT, there is a second variant of the firmware in the :file:`nrf9160dk_asset_tracker_v2_nbiot_legacy_pco_<version-number>.hex` file.
+   Only use this legacy variant if your network does not support ePCO.
+
+#. Click the :guilabel:`Erase & write` button in the :guilabel:`Device` section to program the DK.
+   Do not unplug or turn off the DK during this process.
```

Comment:
This information should come earlier. In Thingy:91 this is a note right at the beginning.